Trimble.Modus.Components/Constant/Constants.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Trimble.Modus.Components.Constant
+{
+    internal class Constants
+    {
+        public const string ButtonEmptyTitleError = "Title cannot be empty";
```

Comment:
I don't think these are needed now. Even if you need them, there is no need to have these as constants.

Reply:
Removed